underfs/s3/src/main/java/tachyon/underfs/s3/S3UnderFileSystem.java

```diff
@@ -84,16 +84,16 @@ public S3UnderFileSystem(String bucketName, TachyonConf tachyonConf) throws Serv
     mBucketName = bucketName;
 
     Jets3tProperties props = new Jets3tProperties();
-    if (tachyonConf.containsKey(Constants.S3_PROXY_HOST)) {
+    if (tachyonConf.containsKey(Constants.UNDERFS_S3_PROXY_HOST)) {
       props.setProperty("httpclient.proxy-autodetect", "false");
-      props.setProperty("httpclient.proxy-host", tachyonConf.get(Constants.S3_PROXY_HOST));
-      props.setProperty("httpclient.proxy-port", tachyonConf.get(Constants.S3_PROXY_PORT));
+      props.setProperty("httpclient.proxy-host", tachyonConf.get(Constants.UNDERFS_S3_PROXY_HOST));
+      props.setProperty("httpclient.proxy-port", tachyonConf.get(Constants.UNDERFS_S3_PROXY_PORT));
     }
-    if (tachyonConf.containsKey(Constants.S3_PROTOCOL)) {
-      props.setProperty("s3service.https-only", tachyonConf.get(Constants.S3_PROTOCOL)
-          .equalsIgnoreCase("http") ? "false" : "true");
+    if (tachyonConf.containsKey(Constants.UNDERFS_S3_PROXY_HTTPS_ONLY)) {
+      props.setProperty("s3service.https-only",
+          tachyonConf.get(Constants.UNDERFS_S3_PROXY_HTTPS_ONLY));
```

Comment:
Done - also needed to add Boolean.toString since props needs a String for the second argument